RELEASE-NOTES.md

```diff
@@ -1,4 +1,33 @@
 ## RELEASE NOTES
+### Version 3.1.5-complex-field-check-answers
+**EUI-4033** Ensured complex fields were interpolated on Check Your Answers page
+
+### Version 3.1.5-pcm-role-non-completion
+**EUI-4077** Ensuring searchForCompletable not called if user is pui-case-manager
+
+### Version 3.1.5 - ph-merge-rc-22-2-a
+**EUI-2027** Accessibility amends for button labels
+**EUI-3155** Accessibility amends for non-descriptive headings
+**EUI-3550** Replace heading tags with spans on case edit pages
+
+### Version 3.1.2-datetime-picker-webapp-resolution
+**EUI-2792** Datetime picker fixes after testing integration with webapp
+
+### Version 3.1.2-missing-ftpa-tab-beta
+**EUI-3825** Fix for missing FTPA tab contents.
```

Comment:
release notes need to be merged to one single version

Reply:
@rayliangatos - sorry, I don't quite follow you... Lines 2 to 17 - these were all (presumably) tags rather than releases; not sure how these notes should be merged to a single version?

Reply:
as agreed this will be refactored when it's actually merged to master